feature-libs/product-configurator/rulebased/components/attribute/types/read-only/configurator-attribute-read-only.component.spec.ts

```diff
@@ -57,27 +74,32 @@ describe('ConfigAttributeReadOnlyComponent', () => {
     expect(component).toBeTruthy();
   });
 
+  it('should test return of extractValuePriceFormulaParameters() method', () => {
```

Comment:
Please rephrase (the method should do something...)

Reply:
done